device/pcie/pci_device.hpp

```diff
@@ -39,15 +50,16 @@ class PCIDevice {
     void write_tlb_reg(uint32_t byte_addr, std::uint64_t value_lower, std::uint64_t value_upper, std::uint32_t tlb_cfg_reg_size);
 
     void open_hugepage_per_host_mem_ch(uint32_t num_host_mem_channels);
-    bool reset_board();
     tt::umd::architecture_implementation* get_architecture_implementation() const { return architecture_implementation.get(); }
 
-    int device_id;
-    int logical_id;
+    PciDeviceInfo info;
+
+    int device_id;  // N in /dev/tenstorrent/N
+    int logical_id; // TODO: does not belong in here
```

Comment:
Is there a difference between these two? I mean, will they evaluate to different values? On a system with two n300, we'll have /dev/tenstorrent/0 and /dev/tenstorrent/1, and then the remote chips will land as 2 and 3, but is it ever possible that logical and pci ids are not the same?

Reply:
I looked closer at the behavior of `create-ethernet-map` and I think under situations most normal users would encounter, they will be the same.  However this is not guaranteed (I can manipulate my system into a state where those two ID values are different).

Reply:
I actually just had this case.
VM gives you /dev/tenstorrent/1, which will resolve to logical_device_id 0 in cluster descriptor, but obviously PCI id will be 1

Reply:
I did not consider the VM or the Docker container case.  Was this with VM on cloud, or IRD?